src/widgets/appointments/appointments-form.component.tsx

```diff
@@ -32,35 +32,58 @@ export default function AppointmentsForm() {
   const [location, setLocation] = useState(null);
   const [serviceUuid, setServiceUuid] = useState("");
   const [serviceTypeUuid, setServiceTypeUuid] = useState(null);
+  const [formChanged, setFormChanged] = useState<boolean>(false);
+  const { t } = useTranslation();
+  let history = useHistory();
+  let providers = null;
```

Comment:
What are providers and status good for?

Reply:
`providers` isn't being used anywhere. `status` is being used in the POST request when creating an appointment though it's still `null` at that point. I can't find docs for the Appointments resource in the REST API docs - perhaps the original author had a reason for putting it there?

Reply:
Ok, but so far I don't see much value in keeping `status` like that.

Can we remove this and just use `status: null` instead of `status: status`? I am easily confused, sorry.

Reply:
Done